bin/slimbookbattery

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
```

Comment:
This should be done project wide, not just in this file.

Reply:
You are right. Anyway, the funny thing is that it is mixed: some files already use this, others use the old...

Reply:
Done.